acceptance_tests/features/steps/reply_to_message_internal.py

```diff
@@ -140,17 +140,21 @@ def url_is_for_my_messages(_):
     assert 'my_conversations=true' in browser.url
 
 
-@given("the secure message user is on page '{page}'")
-def user_on_page(context, page):
-    inbox_internal.go_to_using_context(context)
-
-    for index in range(1, int(page)):
-        browser.driver.find_element_by_class_name('next').click()
-        time.sleep(5)
+@given("the internal user opens first message on page '{page}'")
+def internal_user_opens_first_message_on_page(context, page):
+    internal_user_taken_to_specific_page(context, page)
 
     go_to_thread()
 
 
+@given("the internal user goes to page '{page}'")
+def internal_user_taken_to_specific_page(context, page):
+    inbox_internal.go_to_using_context(context)
+    for page_num in range(1, int(page)):
+        wait_for(named_element_on_page, 5, 1, 'next')
```

Comment:
Whilst you're using this function, I strongly feel like we should be using it in a way that's easier to understand (and hopefully this version proliferate). 
This SHOULD read something like 
`wait_for(named_element_on_page, timeout=5, limit=1, argv='next')`.  It's really really unclear what those 2 numbers mean without going directly to the source so using kwargs here will be helpful.

Reply:
I know what your saying but since the function MAY take a variable number of arguments I cant pass named variables and variable length args in same call in a neat way ? So what I have done is create two helper functions that can take named args which covers us most of the time